modules/appeals_api/spec/sidekiq/higher_level_review_upload_status_batch_spec.rb

```diff
@@ -35,11 +35,9 @@
         let(:upload) { create(:higher_level_review_v1, status: 'received') }
 
         it 'ignores them' do
-          with_settings(Settings.modules_appeals_api, higher_level_review_updater_enabled: true) do
```

Comment:
I believe that unlike the subsequent files, this batch version's `Settings` will need to be replaced with the relevant Flipper flag.

Reply:
@mathisto are you saying you would like me to leave `with_settings...` in the spec?

Reply:
@mathisto Can you be more specific about what changes you want here?